docs/content/en/docs/use-cases/security-profiles/record-linux-capabilities.md

```diff
@@ -0,0 +1,229 @@
+      "binary": "/bin/bash",
+      "flags": "execve rootcwd clone",
+      "start_time": "2023-07-06T10:13:33.178160018Z",
+      "auid": 4294967295,
+      "pod": {
+        "namespace": "default",
+        "name": "xwing",
+        "container": {
+          "id": "containerd://cfb961400ff25811d22d139a10f6a62efef53c2ecc11af47bc911a7f9a2ac1f7",
+          "name": "spaceship",
+          "image": {
+            "id": "docker.io/tgraf/netperf@sha256:8e86f744bfea165fd4ce68caa05abc96500f40130b857773186401926af7e9e6",
+            "name": "docker.io/tgraf/netperf:latest"
+          },
+          "start_time": "2023-07-06T08:07:30Z",
+          "pid": 165
+        },
+        "pod_labels": {
+          "app.kubernetes.io/name": "xwing",
+          "class": "xwing",
+          "org": "alliance"
+        }
+      },
+      "docker": "cfb961400ff25811d22d139a10f6a62",
+      "parent_exec_id": "a2luZC1jb250cm9sLXBsYW5lOjEyODQyMDgxNTA3MzUzOjczODQx",
+      "refcnt": 2,
+      "tid": 73850
+    },
+    "function_name": "cap_capable",
+    "args": [
+      {
+        "user_ns_arg": {
+          "level": 0,
+          "uid": 0,
+          "gid": 0,
+          "ns": {
+            "inum": 4026531837,
+            "is_host": true
+          }
+        }
+      },
+      {
+        "capability_arg": {
+          "value": 34,
+          "name": "CAP_SYSLOG"
+        }
+      }
+    ],
+    "return": {
+      "int_arg": -1
+    },
+    "action": "KPROBE_ACTION_POST"
+  },
+  "node_name": "kind-control-plane",
+  "time": "2023-07-06T10:13:33.834882128Z"
+}
+```
+
+In addition to the Kubernetes Identity and process metadata from exec events, [ProcessKprobe]({{< ref "/docs/reference/grpc-api#processkprobe" >}}) events contain the arguments of the observed system call. In the above case they are:
+
+* `function_name`: that is the `cap_capable` kernel function.
+
+* `user_ns_arg`: is the [user namespace]({{< ref "/docs/reference/grpc-api#usernamespace" >}}) where the capability is required.
+
+   * `level`: is the nested level of the user namespace. Here it is zero which indicates the initial user namespace.
+   * `uid`: is the user ID of the owner of the user namespace.
+   * `gid`: is the group ID of the owner of the user namespace.
+   * [ns]({{< ref "/docs/reference/grpc-api#namespace" >}}): details the information about the namespace. `is_host` indicates that the target user namespace where the capability is required is the host namespace.
+
+* `capability_arg`: is the [capability]({{< ref "/docs/reference/grpc-api#kprobecapability" >}}) required to perform the operation. In this example reading the kernel ring buffer.
+
+   * `value`: is the integer number of the required capability.
+   * `name`: is the name of the required capability. Here it is the `CAP_SYSLOG`.
+
+* `return`: indicates via the `int_arg` if the capability check succeeded or failed. `0` means it succeeded and the access was granted while `-1` means it failed and the operation was denied.
```

Comment:
Usually, I would advise to not put space between list items as it can break markdown listing quite unexpectedly but it seems to work here.